src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -638,6 +639,7 @@ class IrsPlan extends React.Component<
     // generate api calls for each organization id
     const assignmentPromises = organizationIds.map(
       (o: string) =>
+        // tslint:disable-next-line
         new Promise((resolve, reject) => {
```

Comment:
What's the rule?

Reply:
I am going to leave this disabled (but will name the rule) because I am not sure what the function signature should be, and since it is not a test I don't want to use `any`.